datasetIngestor/updateMetaData_test.go

```diff
@@ -0,0 +1,162 @@
+	}
+	if metaDataMap["ownerGroup"] != owner {
+		t.Errorf("Expected ownerGroup to be '%s', got '%s'", owner, metaDataMap["ownerGroup"])
+	}
+	if metaDataMap["endTime"] != endTime {
+		t.Errorf("Expected endTime to be '%v', got '%v'", endTime, metaDataMap["endTime"])
+	}
+	if metaDataMap["license"] != "CC BY-SA 4.0" {
+		t.Errorf("Expected license to be 'CC BY-SA 4.0', got '%s'", metaDataMap["license"])
+	}
+	if metaDataMap["isPublished"] != false {
+		t.Errorf("Expected isPublished to be 'false', got '%v'", metaDataMap["isPublished"])
+	}
+	if _, ok := metaDataMap["classification"]; !ok {
+		t.Errorf("Expected classification to be set, got '%v'", metaDataMap["classification"])
+	}
+}
+
+// Check if updateFieldIfDummy correctly updates the metaDataMap and originalMap when the field is a dummy value.
+func TestUpdateFieldIfDummy(t *testing.T) {
+	// Define test parameters
+	metaDataMap := map[string]interface{}{
+		"testField": "DUMMY",
+	}
+	originalMap := map[string]string{}
+	fieldName := "testField"
+	dummyValue := "DUMMY"
+	newValue := "newValue"
+	
+	// Call the function
+	updateFieldIfDummy(metaDataMap, originalMap, fieldName, dummyValue, newValue)
+	
+	// Check results
+	if metaDataMap[fieldName] != newValue {
+		t.Errorf("Expected %s to be '%v', got '%v'", fieldName, newValue, metaDataMap[fieldName])
+	}
+	if originalMap[fieldName] != dummyValue {
+		t.Errorf("Expected original %s to be '%v', got '%v'", fieldName, dummyValue, originalMap[fieldName])
+	}
+}
+
+// Check if addFieldIfNotExists correctly adds a field to the metaDataMap if it does not already exist.
+func TestAddFieldIfNotExists(t *testing.T) {
+	// Define test parameters
+	metaDataMap := map[string]interface{}{}
+	fieldName := "testField"
+	value := "testValue"
+	
+	// Call the function
+	addFieldIfNotExists(metaDataMap, fieldName, value)
+	
+	// Check results
+	if metaDataMap[fieldName] != value {
+		t.Errorf("Expected %s to be '%v', got '%v'", fieldName, value, metaDataMap[fieldName])
+	}
+}
+
+// Check if updateClassificationField correctly updates the classification field in the metaDataMap.
+func TestUpdateClassificationField(t *testing.T) {
+	// Create a mock server
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte(`[{"TapeRedundancy": "medium", "AutoArchive": false}]`)) // policy list with one policy
+	}))
+	defer ts.Close()
+	
+	// Create a test client
+	client := ts.Client()
+	
+	// Define test parameters
+	APIServer := ts.URL // Use the mock server's URL
+	user := map[string]string{"accessToken": "testToken"}
+	metaDataMap := map[string]interface{}{
+		"ownerGroup": "testOwner",
+	}
+	tapecopies := new(int)
+	*tapecopies = 1
+	
+	// Call the function
+	updateClassificationField(client, APIServer, user, metaDataMap, tapecopies)
+	
+	// Check results
+	if _, ok := metaDataMap["classification"]; !ok {
```

Comment:
can we please also check the value? and potentially if it changes with tape=2? (also, but up to you, would it make sense to have a dedicated function for this: https://github.com/paulscherrerinstitute/scicat-cli/pull/51/files#diff-84473b2b26cf0b84892987a98a715b48ecc9a1e070a19ab030595d1634a12d16R119-R157?)

Reply:
I did that